selftests/unit/test_nrunner.py

```diff
@@ -64,7 +64,7 @@ def test_recipe_exec(self):
                        '"args": ["/etc/profile"], '
                        '"kwargs": {"TERM": "vt3270"}}'))
         with unittest.mock.patch("builtins.open", open_mocked):
-            runnable = nrunner.runnable_from_recipe("fake_path")
+            runnable = nrunner.Runnable.from_recipe("fake_path")
```

Comment:
Ditto here (this deserves to be part of 696eec293b9b36e3afdfdecd55c7e560e4f077db)